src/email/email.js

```diff
@@ -5,21 +5,44 @@ const { Logging } = require('../log');
 
 const log = Logging.get("email");
 
-const roomAlias = function(rcptTo, mxDomain) {
-    let localpartRcptTo;
+// The complete event MUST NOT be larger than 65535 bytes.
```

Comment:
Bit of a tiny nit, but I find it's helpful to link to the source material where possible for the next developer. (In this case https://spec.matrix.org/unstable/client-server-api/#size-limits)